apps/extension/src/side_panel.ts

```
@@ -0,0 +1,141 @@
+  result?: string;
+  timestamp: number;
+}
+
+const queue: QueueEntry[] = [];
+
+function renderQueue(): void {
+  const list = document.getElementById('queueList') as HTMLElement;
+  const empty = document.getElementById('emptyState') as HTMLElement;
+
+  const items = list.querySelectorAll('.queue-item');
+  items.forEach((el) => el.remove());
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)